app/src/components/tags/VerticalTagViewer.vue

```diff
@@ -0,0 +1,89 @@
+<script setup lang="ts">
+import { DocType, db, type TagDto, type Uuid, type queryOptions as options } from "luminary-shared";
+import { computed, ref, watch } from "vue";
+import { useRouter } from "vue-router";
+
+const router = useRouter();
+type Props = {
+    tag?: TagDto;
+    title?: string;
+    queryOptions: options;
+    showPublishDate?: boolean;
+    currentPostId?: Uuid;
+};
+const props = withDefaults(defineProps<Props>(), {
+    showPublishDate: true,
+});
+
+const isContentSelected = (slug: string) => {
+    if (router.currentRoute.value.params.slug === slug) return true;
+    return false;
+};
+
+const taggedDocs = db.contentWhereTagAsRef(props.tag?._id, props.queryOptions);
+
+const currentContent = computed(() => {
+    return taggedDocs.value.filter((doc) => doc._id !== props.currentPostId);
+});
```

Comment:
Why is the current post filtered out here? Don't we want to display it with a highlight? This leads to two weird things on my end:

1. The sidebar looks very empty when there is only one other post in the category
<img width="616" alt="image" src="https://github.com/user-attachments/assets/a80fcd7f-f1bf-4a43-8391-1a1481ea8d71">

2. If the post is the only one within a tag, the whole category is empty.
<img width="561" alt="image" src="https://github.com/user-attachments/assets/269f2d59-f667-4df7-9229-a50291b5f8f5">